massa-models/src/denunciation.rs

```diff
@@ -257,6 +258,18 @@ impl Denunciation {
         }
     }
 
+    /// Check if denunciation has expired given a slot period (assuming the slot is final)
+    pub fn is_expired(
+        denunciation_slot_period: &u64,
+        final_slot_period: &u64,
+        denunciation_expire_periods: &u64,
+    ) -> bool {
+        // If the Slot is final, a Denunciation can still be made for 1 cycle
```

Comment:
remove "1 cycle" from the comment: it is a config parameter

Reply:
done